flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -619,6 +718,21 @@ public ExecutionContext getExecutionContext() {
 			resultStrings[i++] = result;
 		}
 
+		for (Map.Entry<String, List<BlobKey>> entry : largeAccumulatorMap.entrySet()) {
+
+			if(!smallAccumulatorMap.containsKey(entry.getKey())) {
+				StringBuilder str = new StringBuilder();
+				str.append("BlobKeys=[ ");
+				for (BlobKey bk : entry.getValue()) {
+					str.append(bk + " ");
+				}
+				str.append("]");
+
+				StringifiedAccumulatorResult result =
+						new StringifiedAccumulatorResult(entry.getKey(), "Blob/Serialized", str.toString());
+				resultStrings[i++] = result;
+			}
+		}
```

Comment:
Shouldn't this be handled transparently for the user? The user shouldn't have to care if the accumulators were received from the BlobManager or via an Akka message.